### DISCUSSION TITLE
Problem with verifying me student status

### DISCUSSION THREAD

**Discussion by Basiasia21** [community/community#42286] [GitHub Education]
### Select Topic Area

Product Feedback

### Body

I filled up my profile and still I cannot send my application to get a student status, the only info I get is "you're more likely to be verified successfully if you complete your GitHub user profile
please help

**Comment by SilasRodrigues19** [comment]
Hi @Basiasia21 

There is a discussion on this topic at this link: [https://github.com/orgs/community/discussions/30138](https://github.com/orgs/community/discussions/30138)
